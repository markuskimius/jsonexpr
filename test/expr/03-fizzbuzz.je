fizzbuzz = FUNCTION("I",
    x = ARGS[0];

    FOR(i=1, i<x+1, i++,
        IF(i%3 == 0 && i%5 == 0,
            PRINT(i, "fizzbuzz")
        ,i%3 == 0,
            PRINT(i, "fizz")
        ,i%5 == 0,
            PRINT(i, "buzz")
        ,
            PRINT(i)
        )
    );
);

fizzbuzz(30);
