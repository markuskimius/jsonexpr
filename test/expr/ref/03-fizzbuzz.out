1
2
3 fizz
4
5 buzz
6 fizz
7
8
9 fizz
10 buzz
11
12 fizz
13
14
15 fizzbuzz
16
17
18 fizz
19
20 buzz
21 fizz
22
23
24 fizz
25 buzz
26
27 fizz
28
29
30 fizzbuzz
